### PR TITLE
1.37.5

### PR DESCRIPTION
**Pull request by Loloppe**
For new BSML 1.12.0 and BS 1.37.5

### CODE REVIEW

File: Sorters/Models/FolderDateSorter.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ Task Prepare(bool fullReload) {
 					foreach(var song in
 						SongCore.Loader.BeatmapLevelsModelSO
 						._customLevelsRepository?.beatmapLevelPacks.Where(x => x is SongCore.OverrideClasses.SongCoreCustomBeatmapLevelPack)
-						.SelectMany(x => x.beatmapLevels) ?? new List<BeatmapLevel>()
+						.SelectMany(x => x._beatmapLevels) ?? new List<BeatmapLevel>()
```
Comment (Meivyn) [inline Sorters/Models/FolderDateSorter.cs:45]: This should be `.AllBeatmapLevels()`.

**Comment by Meivyn** [comment]
You forgot to update the manifest for BSML/game version.

File: Sorters/Models/FolderDateSorter.cs (hunk 1 above)
Comment (Loloppe) [reply Sorters/Models/FolderDateSorter.cs]: fixed

**Comment by kinsi55** [comment]
Thanks

**Pull request merged**